src/modelinstance.cpp

```diff
@@ -310,8 +310,10 @@ Status ModelInstance::loadOVCNNNetworkUsingCustomLoader() {
         std::string strModel(model.begin(), model.end());
 
         if (res == CustomLoaderStatus::MODEL_TYPE_IR) {
-            network = std::make_unique<InferenceEngine::CNNNetwork>(engine->ReadNetwork(strModel,
-                make_shared_blob<uint8_t>({Precision::U8, {weights.size()}, C}, weights.data())));
+            Blob::Ptr blobWts = make_shared_blob<uint8_t>({Precision::U8, {weights.size()}, C});
+            blobWts->allocate();
```

Comment:
Where do we deallocate this blob?

Reply:
Blob::Ptr is shared pointer, therefore it gets deallocated automatically